code-challenges.test.js

```diff
@@ -23,17 +23,52 @@ var colors1 = ["purple", "blue", "green", "yellow", "pink"]
 var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
-
-
-
+describe("When arrayShuffler is called", () => {
+  it("return ['yellow', 'blue', 'pink', 'green'] if ['purple', 'blue', 'green', 'yellow', 'pink'] is called",() => {
```

Comment:
tests are for documenting behavior, so test descriptions should be generic and describe behavior of function rather than expected inputs and outcome (that's what those expect statements are for)